apps/admin-portal/src/components/applications/forms/inbound-saml-form.tsx

```diff
@@ -0,0 +1,789 @@
+                serviceProviderQualifier: values.get("applicationQualifier"),
+                defaultAssertionConsumerUrl: values.get("defaultAssertionConsumerUrl"),
+                idpEntityIdAlias: values.get("idpEntityIdAlias"),
+                singleSignOnProfile: {
+                    bindings: values.get("bindings"),
+                    enableSignatureValidationForArtifactBinding:
+                        values.get("signatureValidationForArtifactBinding")
+                            .includes("enableSignatureValidationForArtifactBinding"),
+                    attributeConsumingServiceIndex: values.get("attributeConsumingServiceIndex"),
+                    enableIdpInitiatedSingleSignOn: values.get("idPInitiatedSSO").includes("enableIdPInitiatedSSO"),
+                    assertion: {
+                        nameIdFormat: values.get("nameIdFormat"),
+                        audiences: audiences.split(","),
+                        recipients: recipients.split(","),
+                        digestAlgorithm: values.get("digestAlgorithm"),
+                        encryption: {
+                            enabled: values.get("assertionEncryption").includes("enableAssertionEncryption"),
+                            assertionEncryptionAlgorithm: values.get("assertionEncryptionAlgorithm"),
+                            keyEncryptionAlgorithm: values.get("keyEncryptionAlgorithm")
+                        }
+                    }
+                },
+                attributeProfile: {
+                    enabled: values.get("attributeProfile").includes("enabled"),
+                    alwaysIncludeAttributesInResponse: values.get("attributeProfile")
+                        .includes("alwaysIncludeAttributesInResponse")
+                },
+                singleLogoutProfile: {
+                    enabled: values.get("singleLogoutProfile").includes("enabled"),
+                    logoutResponseUrl: values.get("singleLogoutResponseUrl"),
+                    logoutMethod: values.get("logoutMethod"),
+                    idpInitiatedSingleLogout: {
+                        enabled: values.get("idpInitiatedSingleLogout").includes("enabled"),
+                        returnToUrls: returnToURLS.split(",")
+                    }
+                },
+                requestValidation: {
+                    enableSignatureValidation: values.get("requestSignatureValidation")
+                        .includes("enableSignatureValidation"),
+                    signatureValidationCertAlias: values.get("signatureValidationCertAlias")
+                },
+                responseSigning: {
+                    enabled: values.get("responseSigning").includes("enabled"),
+                    signingAlgorithm: values.get("signingAlgorithm")
+                },
+                enableAssertionQueryProfile: values.get("assertionQueryProfile").includes("enableAssertionQueryProfile")
+            }
+        }
+    };
+
+
+    return (
+        <>
```

Comment:
We can further simplify as follows. Wrapping using a fragment and curly braces is not required here.

```tsx
return (
    metadata && (
           ...
    )
)
```